editor/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.9"
+services:
```

Comment:
Can you please add the container_name as well? So that we can map to a particular name if we want to do changes later. 
Also please change the web to app to make it synchronize with the Dockerfile as well. (It should work with that as well, but it would be a good idea to make these names as per our main dockerfile to make it easy for developers to understand)

Reply:
Thanks for your suggestion! Done